zio-http/src/main/scala/zio/http/ZClient.scala

```diff
@@ -350,7 +350,7 @@ trait ZClient[-Env, -In, +Err, +Out] { self =>
     url: String,
     app: SocketApp[Env1],
     headers: Headers = Headers.empty,
-  )(implicit trace: Trace, unsafe: Unsafe): ZIO[Env1 with Scope, Err, Out] =
```

Comment:
It seemed that this was an improper use of `Unsafe`, but please let us know if we're mistaken.